docs/xxhandlesigabrt.md

```diff
@@ -0,0 +1,48 @@
+<!--
+* Copyright (c) 2020, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]HandleSIGABRT
+
+This option affects the handling of the operating system signal `SIGABRT`. This signal represents abnormal termination, and it can either be generated by the `abort` function or the `kill` command.
+
+
+## Syntax
+
+        -XX:[+|-]HandleSIGABRT
```

Comment:
Spacing before the option intentional here?

Reply:
I used [xxhandlesigxfsz.md](https://raw.githubusercontent.com/eclipse/openj9-docs/master/docs/xxhandlesigxfsz.md) as a base template for `xxhandlesigabrt.md`. All other markdown files have this spacing. The spacing is intentional to be consistent with other docs.